src/ruby/yast/rspec/helpers.rb

```diff
@@ -9,60 +9,76 @@ module Helpers
       # When the requested module is present in the system then it is imported
       # as usually. If it is missing a substitute definition is created.
       #
+      # In the substitute definition it is enough to define only the methods
+      # used in the tests, you do not need to mirror the complete API.
+      #
       # @note This needs to be called *after* starting code coverage with
       #   `SimpleCov.start` so the coverage of the imported modules is correctly
       #   counted.
       #
       # @note You can force using the defined stubs although the modules are
       #   present in the system, this might be useful to actually test the stubs.
-      #
-      # @example Mock the `Language` module and the `Language.language` method
-      # Yast::RSpec::Helpers.define_yast_module("Language") do
-      #   # see modules/Language.rb in yast2-country
-      #   module Yast
-      #     class LanguageClass < Module
-      #       # @return [String]
-      #       def language; end
-      #     end
-      #     Language = LanguageClass.new
-      #   end
-      # end
+      #   Just the define the YAST_FORCE_MODULE_STUBS environment variable.
       #
       # @example Mock empty `AutoInstall` module
       # Yast::RSpec::Helpers.define_yast_module("AutoInstall")
       #
+      # @example Mock the `Language` module with the `language` method
+      # Yast::RSpec::Helpers.define_yast_module("Language", methods: [:language])
+      #
+      # @example Mock the `AutoinstStorage` module with `Import` taking one parameter
+      # Yast::RSpec::Helpers.define_yast_module("AutoinstStorage") do
+      #   def Import(_config); end
+      # end
+      #
       # @param name [String] name of the YaST module
-      # @param block [Block] optional module definition, it should provide
-      #   the same API as the original module, it is enough to define only the
-      #   methods used in the tests, if no block is passed an empty module is defined
-      def self.define_yast_module(name, &block)
+      # @param methods [Array<Symbol>] optional list of defined methods,
+      #   the defined methods accept no parameter, if you need a parameter
+      #   then define it in the block
+      # @param force [Boolean] force creating the fake implementation even when
+      #   the module is present in the system, can be also set by the
+      #   YAST_FORCE_MODULE_STUBS environment variable. This might be useful
+      #   if the module constructor touches the system and it would need a lot
+      #   of mocking. But use it carefully, this defeats the purpose of the RSpec
+      #   verifying doubles!
+      # @param block [Block] optional method definitions, they should provide
+      #   the same API as the original module
```

Comment:
I would add note that it can  be combined with methods parameter. So simple methods can be created by param and tricky ones with block. And also that block is basically invoked in context of that class, so it allows attr_readers, constants and other stuff.